content/ripplerest_api.md

```diff
@@ -1721,159 +1419,252 @@ The response is an object with a `orders` array, where each member is a [order o
+        "value": "999"
+      },
+      "taker_pays": {
+        "currency": "USD",
+        "counterparty": "rDZBotqkN4MywSxm9HDtX4m7V6SRkFo7By",
+        "value": "4.74525"
+      },
+      "sequence": 15,
+      "passive": false
     },
-    "taker_pays": {
-      "currency": "BTC",
-      "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B",
-      "value": "21"
+    {
+      "type": "buy",
+      "taker_gets": {
+        "currency": "CAD",
+        "counterparty": "rLr7umFScvEZnj3AJzzZjm25yCZYh3tMwc",
+        "value": "5.35"
+      },
+      "taker_pays": {
+        "currency": "XRP",
+        "counterparty": "",
+        "value": "998.134328"
+      },
+      "sequence": 16,
+      "passive": false
     },
-    "sequence": 106880,
-    "passive": false,
-    "sell": false
-  }, {
-    "taker_gets": {
-      "currency": "BTC",
-      "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B",
-      "value": "21"
+    {
+      "type": "sell",
+      "taker_gets": {
+        "currency": "XRP",
+        "counterparty": "",
+        "value": "999"
+      },
+      "taker_pays": {
+        "currency": "CAD",
+        "counterparty": "rLr7umFScvEZnj3AJzzZjm25yCZYh3tMwc",
+        "value": "5.96403"
+      },
+      "sequence": 17,
+      "passive": false
     },
-    "taker_pays": {
-      "currency": "BTC",
-      "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q",
-      "value": "22.47"
+    {
+      "type": "buy",
+      "taker_gets": {
+        "currency": "CAD",
+        "counterparty": "rLr7umFScvEZnj3AJzzZjm25yCZYh3tMwc",
+        "value": "1103.64"
+      },
+      "taker_pays": {
+        "currency": "USD",
+        "counterparty": "rDZBotqkN4MywSxm9HDtX4m7V6SRkFo7By",
+        "value": "976.6725663716827"
+      },
+      "sequence": 18,
+      "passive": false
     },
-    "sequence": 106066,
-    "passive": false,
-    "sell": false
-  }, {
-    "taker_gets": {
-      "currency": "XRP",
-      "counterparty": "",
-      "value": "102058.710535"
+    {
+      "type": "sell",
+      "taker_gets": {
+        "currency": "USD",
+        "counterparty": "rDZBotqkN4MywSxm9HDtX4m7V6SRkFo7By",
+        "value": "986.89"
+      },
+      "taker_pays": {
+        "currency": "CAD",
+        "counterparty": "rLr7umFScvEZnj3AJzzZjm25yCZYh3tMwc",
+        "value": "1116.17259"
+      },
+      "sequence": 19,
+      "passive": false
     },
+    {
+      "type": "sell",
+      "taker_gets": {
+        "currency": "USD",
+        "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q",
+        "value": "2"
+      },
+      "taker_pays": {
+        "currency": "CAD",
+        "counterparty": "rLr7umFScvEZnj3AJzzZjm25yCZYh3tMwc",
+        "value": "2"
+      },
+      "sequence": 21,
+      "passive": false
+    }
+  ]
+}
+
+```
+
+## Get Order Transaction ##
+[[Source]<br>](https://github.com/ripple/ripple-rest/blob/59ea02d634ac4a308db2ba21781efbc02f5ccf53/api/orders.js#L505 "Source")
+
+Get the details of an order transaction.
```

Comment:
An order transaction is an `OfferCreate` or `OfferCancel`. The notifications API is now pointing to this endpoint (previously it would point to /v1/transactions/{:hash} which just returned the raw data from rippled.

I used order "transaction" because it's not an order (`OfferEntry`) on the ledger / orderbook. It's a transaction that could create or consume these.

I can add language to make this distinction. Perhaps calling it "Get Order Submission" would be clearer?